src/tdlib/message_interaction_info.rs

```diff
@@ -0,0 +1,74 @@
+use gtk::glib;
+use gtk::prelude::*;
+use gtk::subclass::prelude::*;
+use tdlib::types;
+
+mod imp {
+    use super::*;
+    use once_cell::sync::Lazy;
+    use std::cell::Cell;
+
+    #[derive(Debug, Default)]
+    pub(crate) struct MessageInteractionInfo {
+        pub(super) reply_count: Cell<u32>,
+    }
+
+    #[glib::object_subclass]
+    impl ObjectSubclass for MessageInteractionInfo {
+        const NAME: &'static str = "MessageInteractionInfo";
+        type Type = super::MessageInteractionInfo;
+    }
+
+    impl ObjectImpl for MessageInteractionInfo {
+        fn properties() -> &'static [glib::ParamSpec] {
+            static PROPERTIES: Lazy<Vec<glib::ParamSpec>> = Lazy::new(|| {
+                vec![glib::ParamSpecUInt::builder("reply-count")
+                    .read_only()
+                    .build()]
+            });
+            PROPERTIES.as_ref()
+        }
+
+        fn property(&self, _id: usize, pspec: &glib::ParamSpec) -> glib::Value {
+            match pspec.name() {
+                "reply-count" => self.obj().reply_count().to_value(),
+                _ => unimplemented!(),
+            }
+        }
+    }
+}
+
+glib::wrapper! {
+    pub(crate) struct MessageInteractionInfo(ObjectSubclass<imp::MessageInteractionInfo>);
+}
+
+impl From<Option<types::MessageInteractionInfo>> for MessageInteractionInfo {
+    fn from(interaction_info: Option<types::MessageInteractionInfo>) -> Self {
+        let obj: Self = glib::Object::builder().build();
+        obj.update(interaction_info);
```

Comment:
This will make the properties to "notify" on construction. I'd avoid that because there's a cost on notifying. You can just set the `reply_count`.